build.gradle

```diff
@@ -67,6 +67,7 @@ subprojects {
     apply plugin: 'java'
     apply plugin: 'maven'
     apply plugin: 'signing'
+    apply plugin: 'maven-publish'
```

Comment:
The plugin published artifacts to the local m2 repo on the machine. For some reason it no longer works. If you go back to commit 4564e65 and do a `./gradlew clean assemble publishToMavenLocal` it will push all artifacts to the local m2. The task `publishToMavenLocal` is brought in by `maven-publish` plugin. I would hold off on merging this until we can figure out why artifacts are not being published with this task. I thought this would fix it, but further testing shows it does not.